src/buffer/out/TextAttribute.cpp

```diff
@@ -365,10 +375,11 @@ void TextAttribute::SetDefaultBackground() noexcept
 }
 
 // Method description:
-// - Resets only the rendition character attributes
+// - Resets only the rendition character attributes, which includes everything
+//     except the Protected attribute.
 void TextAttribute::SetDefaultRenditionAttributes() noexcept
 {
-    _attrs = CharacterAttributes::Normal;
+    _attrs &= CharacterAttributes::Protected;

```

Comment:
Should we introduce a metaconstant for the rendition attributes? or should we just defer that until we have more non-rendition attributes... :)

Reply:
I don't know. If we did, it would be everything but `Protected`, would seems like a pain to maintain. We could go the other way around and create a metaconstant for the "logical" attributes, but that would just be `Protected` which feels a bit pointless. I'd say defer it until have more non-rendition attributes, but realistically that'll probably be never.